CHANGELOG.md

```diff
@@ -1,34 +1,181 @@
-| Added venia-concept as a dependency of create-pwa | **Bugfix**  | [#3008][] |
+| Description                                                            | Change type | PR        |
+| ---------------------------------------------------------------------- | ----------- | --------- |
+| Added ability to use custom template for scaffolding                   | **Feature** | [#3025][] |
+| Added option to use custom https certificates with buildpack           | **Feature** | [#2946][] |
+| Updated compile time logic to fetch store name from the GraphQL server | **Update**  | [#3019][] |
+| Fixed command escaping in the `create-pwa` shell command               | **Bugfix**  | [#3022][] |
+
+### UPWARD
+
+| Description                                                     | Change type | PR        |
+| --------------------------------------------------------------- | ----------- | --------- |
+| Added support for express server compression middleware.        | **Feature** | [#2980][] |
+| Added ability to allow `blob` types for UPWARD security headers | **Feature** | [#2985][] |
+| Added the xfwd option to the proxyMiddleware                    | **Feature** | [#2986][] |
+
+### Extensions
+
+| Description                                           | Change type | PR        |
+| ----------------------------------------------------- | ----------- | --------- |
+| Added a null check for element styling in PageBuilder | **Update**  | [#3016][] |
+
+### Documentation
+
+| Description                                     | Change type       | PR        |
+| ----------------------------------------------- | ----------------- | --------- |
+| Published a new tag list extension tutorial     | **Documentation** | [#3044][] |
+| Published new topic about extension development | **Documentation** | [#2995][] |
+| Published new topic about Targetables           | **Documentation** | [#2966][] |
+| Updated code sample in documentation            | **Update**        | [#2938][] |
+| Updated getting started section in docs         | **Update**        | [#2926][] |
+| Updated "Modify talon results" tutorial         | **Bugfix**        | [#3039][] |
+| Fixed broken link in doc topic                  | **Bugfix**        | [#3002][] |
+
+### Misc
+
+| Description                                  | Change type | PR        |
+| -------------------------------------------- | ----------- | --------- |
+| Removed auto-assign logic for new docs issue | **Update**  | [#3040][] |
+| Upgraded workbox to v6                       | **Update**  | [#2983][] |
+| Added coveralls badge back to the README     | **Update**  | [#2978][] |
+| Upgraded React to v17                        | **Update**  | [#2962][] |
+| Enabled singleton style tag                  | **Update**  | [#2906][] |
+| Updated prettier config                      | **Update**  | [#2900][] |
+| Removed cyclic dependencies                  | **Bugfix**  | [#2967][] |
 
 ## Known issues
 
```

Comment:
GraphQL issue https://jira.corp.magento.com/browse/PWA-1298 is fixed on 2.4-develop, good to add this note/info info if customers like to get patch.